bookwyrm/static/css/bookwyrm.css

```diff
@@ -72,6 +72,10 @@ body {
     flex-grow: 1;
 }
 
+.preserve-whitespace p {
+    white-space: pre-wrap !important;
```

Comment:
Still unsure if it would be best to use pre-wrap here or break-spaces instead

Reply:
I had to look up what the difference was! I think pre-wrap is better